openapi/src/parts/controlled_azure_storage.yaml

```diff
@@ -24,6 +24,28 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/storageContainer/{resourceId}/getSasToken:
```

Comment:
I would put this in controlled_azure_storage_container.yaml instead given that the URL path includes `storageContainer`.